components/org.wso2.carbon.identity.auth.service/src/main/java/org/wso2/carbon/identity/auth/service/AuthenticationContext.java

```diff
@@ -59,6 +62,16 @@ public void setUser(User user) {
         this.user = user;
     }
 
+    public Object getProperties(String key) {
+
+        return properties.get(key);
+    }
+
+    public void setProperties(String key, Object value) {
```

Comment:
since we are only adding a single property to the poperty map, shall we rename the method as addToProperties?